silkworm/common/log.hpp

```diff
@@ -0,0 +1,48 @@
+/*
+   Copyright 2020 The Silkworm Authors
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+#ifndef SILKWORM_COMMON_LOG_H_
+#define SILKWORM_COMMON_LOG_H_
+
+#include "tee.hpp"
```

Comment:
Pls #include <silkworm/common/tee.hpp> instead

Reply:
It won't build that way.  I was following the example of util.cpp.

Reply:
It should build - other headers include silkworm headers as <silkworm/...>

Reply:
It should build - other headers include silkworm headers as <silkworm/...>